mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionDaoImpl.java

```diff
@@ -42,166 +58,216 @@
   private static final String TABLE_NAME = "job_executions";
   private static final String ID_FIELD = "'id'";
   public static final String GET_JOB_EXECUTION_HR_ID = "SELECT nextval('%s.job_execution_hr_id_sequence')";
+  public static final String GET_JOBS_WITHOUT_PARENT_MULTIPLE_QUERY_PATH = "templates/db_scripts/get_job_execution_without_parent_multiple.sql";
+  public static final String TOTAL_ROWS_COLUMN = "total_rows";
+  private static final String JSONB_COLUMN = "jsonb";
+
+  private String getJobsWithoutParentMultipleSql;
 
   @Autowired
   private PostgresClientFactory pgClientFactory;
 
+  @PostConstruct
+  public void init() {
+    try {
+      getJobsWithoutParentMultipleSql = ResourceUtil.asString(GET_JOBS_WITHOUT_PARENT_MULTIPLE_QUERY_PATH);
+    } catch (UncheckedIOException e) {
+      LOGGER.error("Error while reading query from file {}", GET_JOBS_WITHOUT_PARENT_MULTIPLE_QUERY_PATH, e);
+      throw e;
```

Comment:
What should be in case of throw?

Reply:
spring context initialization and verticle deployment will fail

Reply:
isn't it too harsh? I'd prefer for the app to return 500 error on some of the requests than failing on startup, leading to the whole data-import functionality to fail